requirements.txt

```diff
@@ -0,0 +1,3 @@
+# The "." states that pip should look at the setup.py
+# for requirements.
+.
```

Comment:
See comment in `Dockerfile`

Reply:
This is useful if you like to install the deps of the package without installing the package itself.